src/layers/WellboreBaseComponentLayer.ts

```diff
@@ -200,4 +203,8 @@ export class WellboreBaseComponentLayer extends PixiLayer {
 
     return this._textureCache[cacheKey];
   }
+
+  get renderType(): RENDERER_TYPE {
```

Comment:
Sadly the grammar for getters does not allow for arrow expressions. Would have been much prettier. I know that this comment does not give much value, but that was my thought after looking at this method... 🙃